lighthouse-core/gather/gatherers/seo/embedded-content.js

```diff
@@ -5,36 +5,42 @@
  */
 'use strict';
 
+/* globals getElementsInDocument */
+
 const Gatherer = require('../gatherer.js');
 const pageFunctions = require('../../../lib/page-functions.js');
 
+function getEmbeddedContent() {
+  const selector = 'object, embed, applet';
+  /** @type {HTMLElement[]} */
+  // @ts-expect-error - getElementsInDocument put into scope via stringification
```

Comment:
nit: can these comments be combined?

Reply:
Maybe, but we do this in quite a few places.

Reply:
Fair enough, not worth the effort to find out

Reply:
> nit: can these comments be combined?

the jsdoc parser has improved and lets you combine them (it didn't previously), but `@ts-expect-error` still has to be first, so it's some permutation of
```js
/** @ts-expect-error - getElementsInDocument put into scope via stringification @type {HTMLElement[]} */
const elements = getElementsInDocument(selector);
```
or
```js
/** @ts-expect-error @type {HTMLElement[]} - getElementsInDocument put into scope via stringification */
const elements = getElementsInDocument(selector);
```

Personally I don't think it improves readability but YMMV :)

Reply:
(also `@ts-expect-error - getElementsInDocument put into scope via stringification` ends up as a docstring on hover of the variable :)